test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -122,7 +108,7 @@ class ConsensusSunnyDayTest : public ::testing::Test {
     }
     if (num_peers == 1) {
       delay_before = 0;
-      delay_after = 50;
+      delay_after = 5 * 1000;
```

Comment:
Why was delay changed? So, in this case, we also use cv and this is kinda of time assumption.

Reply:
Time assumption is required because gRPC is used in the test. Seems like 50 milliseconds was not enough and caused the test to fail from time to time.